gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,128 @@
+---
+id: gigamon
+backend_only: true
+facets:
+  - facetType: list
+  - groups:
```

Comment:
This isn't a right format, you don't need the `-` before groups as this makes it a new item in the `facets` list.

Reply:
removed "-facetType:list" line and it worked well.